src/models/converters/transactions/summary.rs

```diff
@@ -39,7 +39,7 @@ impl MultisigTransaction {
     pub fn to_transaction_summary(
         &self,
         info_provider: &mut dyn InfoProvider,
-    ) -> Result<Vec<TransactionSummary>> {
+    ) -> ApiResult<Vec<TransactionSummary>> {
```

Comment:
I am not sure there was any advantage to being able to use `anyhow::Result`. `ApiResult` was a type that I personally considered to be used when interecting with the clients (to expose outwards). I suppose we don't need that distinction though.

Reply:
`anyhow::Result` is a wrapper around `Result` (with a default error type)  and our `ApiResult` was a wrapper around that with another error type. 

Because of the double wrapping weird back and forth conversion was happening that resulted in `404` being converted to `500`. 

While there is propbably a way to use `anyhow::Result` everywhere it seemed more complex, as we in most of our app already use `ApiResult` and have the required helper methods.